.idea/runConfigurations/_template__of_JavaScriptTestRunnerJest.xml

```diff
@@ -0,0 +1,9 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="true" type="JavaScriptTestRunnerJest">
+    <node-interpreter value="project" />
+    <node-options value="--experimental-vm-modules" />
+    <envs />
+    <scope-kind value="ALL" />
+    <method v="2" />
+  </configuration>
+</component>
```

Comment:
should this be in this PR?

Reply:
No, it must've been added automatically when I merged the PR this one is based on.

BTW, this is a draft PR, so not ready to be reviewed yet.

Reply:
This is gone now that I've merged `main`.